### PR TITLE
WIP: Update rails

### PR DESCRIPTION
**Pull request by pjvv**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ default: &default
 
 development:
   <<: *default
-  database: submission_service_development
```
Comment (NikolausC) [inline config/database.yml]: Why have these changed?

Comment (pjvv) [reply config/database.yml]: To correct the name..

Comment (NikolausC) [reply config/database.yml]: Oh yeah 🙈

File: config/environments/development.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Verifies that versions and hashed value of the package contents in the project's package.json
```
Comment (NikolausC) [inline config/environments/development.rb]: What's this comment trying to say? 🤔

Comment (pjvv) [reply config/environments/development.rb]: Not sure, comment came with the upgrade..

File: spec/features/taxon_id_finder_spec.rb.TO_FIX [hunk 3]
```diff
@@ -72,6 +72,8 @@
         context 'when writing a taxonomy id in a single input' do
           it 'uses the taxonomy service and searches for the id provided', js: true  do
             fill_in('labware[1]address[A:1]fieldName[taxon_id]', with: '3')
+            puts current_url
+            binding.pry
```
Comment (NikolausC) [inline spec/features/taxon_id_finder_spec.rb.TO_FIX]: Whoops!

**Comment by NikolausC** [comment]
Don't think we can test this until we sort the RVM installation issue

**Comment by pjvv** [comment]
Stalling for now.. More work needs to be done on the JS files after the move to webpacker. The CSV checker in particular does not seem to be behaving with webpacker.

**Pull request merged**